e2e/cypress/integration/scroll/fixed_width_spec.js

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+const timeouts = require('../../fixtures/timeouts');
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @scroll
+describe('Scroll', () => {
+    let testTeam;
+    let testChannel;
+
+    before(() => {
+        cy.viewport(1800, 2000);
```

Comment:
0/5 to set such a high view port unless its absolutely necessary. Currently the test works fine even without setting this view port. Hence can we remove this.

Reply:
@srkgupta thanks, 
that's exactly my observation was.
I have reverted View-port